SpatialGDK/Source/SpatialGDK/Public/SpatialConstants.h

```diff
@@ -23,29 +23,30 @@ enum ESchemaComponentType : int32
 	SCHEMA_OwnerOnly,
 	SCHEMA_Handover,
 
+	SCHEMA_Count,
```

Comment:
Should this value be updated to indicate it's only including data components? Where is it even being used?